scripts/jest/src/v0/setupTests.js

```diff
@@ -16,11 +16,4 @@ if (process.env.TF_BUILD) {
   jest.spyOn(console, 'info');
   jest.spyOn(console, 'warn');
   jest.spyOn(console, 'error');
-
-  afterAll(() => {
```

Comment:
this will is no longer as we reset mocks automatically to avoid hidden leaks -> all of these are not spies anymore during `afterAll` teardown

Reply:
As spies don't get called, you can remove the whole block in the if condition

Reply:
yeah I wanted to do that but opted out for unknown reasons  :D

Reply:
done